pkg/connector/events.go

```diff
@@ -340,9 +340,13 @@ func (evt *WAUndecryptableMessage) ConvertMessage(ctx context.Context, portal *b
 	}
 	content := &undecryptableMessageContent
 	if evt.Type == events.UnavailableTypeViewOnce {
+		verb := "received"
+		if evt.Info.IsFromMe {
```

Comment:
i forget if `IsFromMe` is always accurate, like would this be `true` if another device relayed us someone else's message?

Reply:
It is always accurate, responses to message requests from phone are wrapped in a `WebMessageInfo` in order to have accurate event metadata